meta-lhg-integration/recipes-kernel/linux/linux-hikey/0001-staging-ion-add-Allwinner-ION-secure-heap-not-for-ma.patch

```diff
@@ -0,0 +1,227 @@
+From e31dd54997b050b6a6965d7cfbc795492256847c Mon Sep 17 00:00:00 2001
+From: Etienne Carriere <etienne.carriere@linaro.org>
+Date: Thu, 23 Mar 2017 14:02:08 +0100
+Subject: [PATCH 1/3] staging/ion: add Allwinner ION "secure" heap **not for
+ mainline**
+
+Dumped from:
+  https://github.com/loboris/OrangePI-Kernel/tree/master/linux-3.4
+  0cc8d855adb457d1860d6e25cb93b6cc75d5a09d
+  Author: Sunny <sunny@allwinnertech.com> for Allwinner.
+
+Changes made on original "secure heap" implementation:
+- minor coding style: fix includes, empty lines and overlong lines,
+  indentation, comment layout.
+- Original path modified the ion uapi. We do not attempt to modify
+  uapi/ion.h. "secure" (or "domain") heaps are under ID
+  ION_HEAP_TYPE_CUSTOM + 1 (legacy 'secure heap type' value).
+
+Signed-off-by: Etienne Carriere <etienne.carriere@linaro.org>
+Reviewed-by: Joakim Bech <joakim.bech@linaro.org>
+Signed-off-by: jennifer111 <jie.mu@linaro.org>
```

Comment:
You should not sign-off patches taken as is from somewhere (you are not the author of the patch, did not participate in the discussion, aren't the maintainer of the tree the patch was accepted into).